base/deprecated.jl

```diff
@@ -268,5 +268,9 @@ end
 
 @deprecate var"@_inline_meta"   var"@inline"   false
 @deprecate var"@_noinline_meta" var"@noinline" false
+@eval function getindex(@nospecialize(t::Tuple), i::Real)
+  Base.depwarn("`getindex(t::Tuple, i::Real)` is deprecated, use `t[convert(Int, i)]` instead.", ((Base.Core).Typeof(getindex)).name.mt.name)
```

Comment:
Why not just
```suggestion
  Base.depwarn("`getindex(t::Tuple, i::Real)` is deprecated, use `t[convert(Int, i)]` instead.", :getindex)
```
?

Reply:
because I was looking at the `@macroexpand` output of `@deprecate`